07-graphs/__test__/graph.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Graph = require('../lib/graph_traverse');
+
+describe('tests for breadth first traversal', () => {
+  test('creates instance of a graph', () => {
+    let graph = new Graph();
+    expect(graph).toBeInstanceOf(Graph);
+  });
+  test('creates instance of a graph with a new vertex', () => {
+    let graph = new Graph();
+    graph.addVertex('b', ['a', 'c', 'd']);
+
+    expect(graph.vertices['b']).toEqual(['a', 'c', 'd']);
+  });
+
+  test('will have neighbors when value is added', () => {
```

Comment:
This test and the test above it are testing the same condition.
Although they have different `test` descriptions, they are actually the same.